typescript/introduction/tsc-interfaces-classes/tsc-classes-intro.md

```diff
@@ -0,0 +1,77 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Classes Intro
+
+---
+## Content
+
+Classes represent the building blocks of the style of programming called *object-oriented (class-based) programming*, or OOP[1]. 
+
+The idea behind it is to organize related values and behaviors into definitions of entities that exist in your program (an "Animal", a "GamePlayer", a "User", etc). Then you can use these definitions, called classes, to *create (or instantiate)* those entities wherever you'd like. These instances of classes are called objects.
+
+To define a class, TypeScript gives us the `class` keyword. To create an instance of that class we can use the `new` keyword. Let's look at an example:
```

Comment:
I think this is JS syntax that's extended by typescript to allow adding types to all class variables

seems we present it like "oh this is something new in and specific to TS"

Reply:
The thing is, not all people learning TypeScript already know JavaScript. Someone might encounter all of this the first time in TypeScript.

Reply:
same can be said about array, floats etc. my point is we shouldn't say classes are an inherent ts feature